Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2429-cleanup-eua

Resolves #EREGCSC-2429

**Description-**
Refactor the eua_settings.py file. During the development phase of the EUA integration, the eua_settings.py file was utilized for local and development environments. As the integration progressed to production, the variables employed by eua_settings.py were duplicated into local.py, deploy.py, and test.py. Consequently, eua_settings.py became obsolete. To streamline the configuration files on the production environment, an additional step is taken for cleanup.

Since the values across environments remain consistent, duplicating them across multiple files becomes unnecessary and complicates file maintenance. Consequently, eua_settings.py is now designated to house all environment-specific variables related to authentication and EUA. Any settings file requiring these values can simply import eua_settings.py, resulting in a more organized setup and a file that is easier to manage.


**This pull request changes...**

- expected change 1

**Steps to manually verify this change...**

Steps to view and verify change
-  sign into /admin using eua, ensure authentication process works. 
-  make sure all the tests pass
-  ensure that the code deploys to experimental with no issues.
